docs/diagrams/MarkPSequenceDiagram.puml

```diff
@@ -0,0 +1,58 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":TaskHubParser" as TaskHubParser LOGIC_COLOR
+participant "c:MarkProjectCommand" as MarkProjectCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+[-> LogicManager : execute("markP 1 3")
+activate LogicManager
+
+LogicManager -> TaskHubParser : parseCommand("markP 1 3")
+activate TaskHubParser
+
+create MarkProjectCommand
+TaskHubParser -> MarkProjectCommand
+activate MarkProjectCommand
+
+MarkProjectCommand --> TaskHubParser
+deactivate MarkProjectCommand
+
+TaskHubParser --> LogicManager : c
```

Comment:
Does the `c` stand for command? If so, then maybe future sequence diagrams describing our implementations of commands should follow suit. Existing ones should also be changed to match.

Reply:
Yes, `c` stands for command. I have standardized it now.